pom.xml

```diff
@@ -913,7 +918,6 @@
                                 <version>${version.maven}</version>
                             </requireMavenVersion>
                             <requirePluginVersions />
-                            <requireSameVersionsReactor />
```

Comment:
It sounds similar to the `ReactorModuleConvergence`.

Reply:
Yep. Unfortunately `ReactorModuleConvergence` is [not compatible](https://maven.apache.org/enforcer/enforcer-rules/reactorModuleConvergence.html) with `-pl`, so not something we can enable.